lib/interface_quda.cpp

```diff
@@ -523,6 +523,10 @@ void initQuda(int dev)
 // possible flag to indicate we need to recompute the clover field
 static bool invalidate_clover = true;
 
+// pre-declare some cleanup utilities
+void freeUniqueSloppyGaugeUtility(cudaGaugeField*&, cudaGaugeField*&, cudaGaugeField*&, cudaGaugeField*&, cudaGaugeField*&);
+void freeUniqueGaugeUtility(cudaGaugeField*&, cudaGaugeField*&, cudaGaugeField*&, cudaGaugeField*&, cudaGaugeField*&, cudaGaugeField*&, QudaBoolean);
+
```

Comment:
Wonder where are these two functions implemented?

Reply:
Also might be worth to give them a doxygen.

Reply:
Haha, yes, good catch. They're implemented further down (closer to the other `free` functions), but they also have utility for the `load` functions.

Reply:
https://github.com/lattice/quda/pull/1362/commits/02950b82682235b89f67a80b762b1d62986535ce